### PR TITLE
Add `JsonLd` docs

### PR DESCRIPTION
**Pull request by klimov-paul**
Relates to #141

Adds docs for `JsonLd` usage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,71 +12,111 @@ SEOTools is a package for **Laravel 5+** and **Lumen** that provides helpers for
 
 [![Latest Stable Version](https://poser.pugx.org/artesaos/seotools/v/stable)](https://packagist.org/packages/artesaos/seotools) [![Total Downloads](https://poser.pugx.org/artesaos/seotools/downloads)](https://packagist.org/packages/artesaos/seotools) [![Latest Unstable Version](https://poser.pugx.org/artesaos/seotools/v/unstable)](https://packagist.org/packages/artesaos/seotools) [![License](https://poser.pugx.org/artesaos/seotools/license)](https://packagist.org/packages/artesaos/seotools)
 
-## Features
-- Friendly Interface
-- Ease of set titles and meta tags
-- Ease of set metas for twitter and opengraph
+Features
+--------
```
Comment (antonkomarev) [inline README.md]: Why change H2 heading definition style? (here and further)

File: README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
-# SEOTools - SEO Tools for Laravel and Lumen
+SEOTools - SEO Tools for Laravel and Lumen
+==========================================
```
Comment (antonkomarev) [inline README.md]: Why change H1 heading definition style?

File: README.md (hunk 1 above)
Comment (klimov-paul) [reply README.md]: This does not change document HTML representation, but significantly improves plain-text representation making source '.md' more readable.

File: README.md (hunk 2 above)
Comment (klimov-paul) [reply README.md]: This does not change document HTML representation, but significantly improves plain-text representation making source '.md' more readable.

Comment (antonkomarev) [reply README.md]: It only adds inconsistency with h3 headings, IMO.

Comment (klimov-paul) [reply README.md]: I prefer better source code comprehension to such kind of consistency.
But you are free to switch definition back if you like.
Just don't expect my help for it - I have wasted enough time on this already.

**Pull request merged**